docs/release/release_0_4_19.md

```diff
@@ -0,0 +1,350 @@
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://napari.org/stable/
+
+## Highlights
+
+This release is mostly a bug fix release. We aim for the next release to be 0.5.0, with many new exciting features!
+
+Highlights in `v0.4.19`:
+
+A new feature: [napari/napari#6102](https://github.com/napari/napari/pull/6102) added convenient parsing of color names and hex color strings in layer data
+and generating black-to-color colormaps when adding layers to the viewer. 
+For example, one can now use: `viewer.add_image(data, colormap="crimson")` or `viewer.add_image(data, colormap="#FAFA00").
+To see what color names can be handled, please refer to the 
+[vispy` dictionary of colors](https://github.com/vispy/vispy/blob/269ed1ac4d8126421fd5a7eb06a2996d63f46b17/vispy/color/_color_dict.py#L181).
+Note: there is a namespace collision for the case of `orange` which is both a proper `vispy` colormap name and a color name. 
+Previously, `orange` was handled as the `vispy` colormap, which is from white to orange. 
+However, with 0.4.19, `orange` will be handled as a color and the generated colormap will be from black to orange. 
+
+The removal of public access to `viewer.window.qt_viewer` has been postponed to `v0.6.0` ([napari/napari/#6283](https://github.com/napari/napari/pull/6283)).
+If you're interested in some feature that's currently behind private or deprecated API,
+please make an issue on the napari repo requesting the feature to be exposed!
+
+napari is now pydantic 2 compatible. At this moment we are using `pydantic.v1` to achieve this. 
+Additionally, for this release, the napari conda bundle will be shipped with pydantic v1, but
+we expect that in the next release we will ship the bundle with pydantic v2.
+Please, if you are a plugin developer and your plugin uses pydantic, ensure
+that it is compatible with pydantic v2 ([napari/napari/#6358](https://github.com/napari/napari/pull/6358)).
+
+We have fixed problems with rendering some labels in incorrect color by sequence of Pull Requests:
+[napari/napari/#3308](https://github.com/napari/napari/pull/3308),
+[napari/napari/#6411](https://github.com/napari/napari/pull/6411),
+[napari/napari/#6467](https://github.com/napari/napari/pull/6467),
+[napari/napari/#6439](https://github.com/napari/napari/pull/6439).
+[napari/napari/#6607](https://github.com/napari/napari/pull/6607).
+This also improves performance of rendering labels.
+However, there are still some limitations.
+Best performance is for labels encoded as (u)int8.
+As GPU API accept 8 and 16 bits int textures and float textures,
+for labels stored as (u)int8 and (u)int16 we could transfer data to GPU without preprocessing.
```

Comment:
```suggestion
As the GPU API accepts 8 and 16 bits int textures and float textures,
for labels stored as (u)int8 and (u)int16 we could transfer data to GPU without preprocessing.
```

Reply:
could yo make remarks here https://github.com/napari/napari-release-tools/pull/3/files#diff-f175a32e4fcbdd7db7a328d3c6d0e2a6092f89a3480917f8e1d0fe50122ff026

Reply:
Yes I will after meeting

Reply:
Sorry, missed this. Will also move mine there.